sdk/search/azure-search-documents/README.md

```diff
@@ -235,6 +235,10 @@ These are just a few of the basics - please [check out our Samples][samples_read
 * [Retrieving a specific document from your index](#retrieving-a-specific-document-from-your-index)
 * [Async APIs](#async-apis)
 
+### Advanced authentication
```

Comment:
nit: we don't need a separate sub-section for this. It can just be added to the existing list of examples.